externs/webos.js

```diff
@@ -0,0 +1,19 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Externs for WebOS
+ *
+ * @externs
+ */
+
+
+/** @const */
+var PalmSystem = {};
+
+
+/** @type {string} */
+PalmSystem.deviceInfo;
```

Comment:
I wasn't aware of this API. And it's hard to find anything about it.
Personally I'd use [webOS.js API](https://webostv.developer.lge.com/develop/references/webostvjs-webos#deviceinfo) because it's well described.

Reply:
True, but the webos.js api inside uses this api, and we also don't need the user to load webos.js to be able to detect the resolution correctly :)

Reply:
https://webos-devrel.github.io/webOS.js/deviceinfo.js.html

Reply:
If it uses it internally, then sure, keep it simple!

Reply:
@tykus160 I see that you are interested in this PR, would you be interested in review the rest of the platforms if you use them?

Reply:
Sure, let me compare it with what we use internally.